src/components/Modal.js

```diff
@@ -74,9 +74,9 @@ const Modal = ({ onDismiss, title, titleExtras, children, width = 450, showCance
         }, [cancelText]) :
         null,
       Utils.cond(
-        [okButton === undefined, () => h(ButtonPrimary, { onClick: onDismiss }, 'OK')],
-        [_.isString(okButton), () => h(ButtonPrimary, { onClick: onDismiss }, okButton)],
-        [_.isFunction(okButton), () => h(ButtonPrimary, { onClick: okButton }, 'OK')],
+        [okButton === undefined, () => h(ButtonPrimary, { onClick: onDismiss, danger }, 'OK')],
+        [_.isString(okButton), () => h(ButtonPrimary, { onClick: onDismiss, danger }, okButton)],
+        [_.isFunction(okButton), () => h(ButtonPrimary, { onClick: okButton, danger }, 'OK')],
```

Comment:
I like this, but consider whether it can be omitted for the sake of preserving existing files as much as possible (when introducing new functionality). This would be a great addition as a follow-up PR.

Reply:
This particular change allowed me to make the Abort Upload button red instead of blue, which more accurately captures its purpose (don't press that button unless you really mean it). However, I'd be happy to pull that out into a different PR for cleanliness.

Reply:
Tough call, probably should stay in this PR.